tailwind.config.ts

```diff
@@ -130,8 +130,9 @@ export default {
         xs: '0px 1px 2px 0px theme(colors.shadow / 5%)',
         lg: '0px 4px 6px -2px theme(colors.shadow / 3%), 0px 12px 16px -4px theme(colors.shadow / 8%)',
       },
-      spacing: { '4.5': '1.125rem' },
+      spacing: { '4.5': '1.125rem', '18': '4.5rem' },
```

Comment:
We don't necessarily have to have 4.5rem, right? What happens if we use the closest `px` for this value? How different it looks?

Reply:
The required spacing is 72px(4.5rem), I can add a spacing of 64px(4rem) or 80px(5rem) instead

64px / 4rem;
<img width="1710" alt="image" src="https://github.com/nodejs/nodejs.org/assets/14062599/29a8d44f-d5cb-4c4e-8e7f-59591461c414">

80px / 5rem;
<img width="1710" alt="image" src="https://github.com/nodejs/nodejs.org/assets/14062599/581c03d8-239c-4c65-8cda-4704f4bd4cb6">

IMO 64px one looks good, and hard to tell difference 👀